src/components/Card.vue

```diff
@@ -61,7 +61,7 @@ export default {
       this.height = $refs.wrapper.offsetHeight;
       // Adjust the number of rows the card spans (necessary for the masonry layout to work)
       this.spanValue = Math.ceil((this.height + margin * 2) / 5); // 5 is row height
-      this.$emit('height-change');
+      // this.$emit('height-change');
```

Comment:
Thanks for bringing this up The issue I was encountering is that the `ScheduleCard.vue` would call `scrollToCurrentPeriod()` every time the interval function was called. This created the unwanted behavior of it making it impossible to scroll up or down because it would call `scrollToCurrentPeriod` and bring you back to the current period.   

Commenting that line was the temporary solution I found. I will look more into what I can do to fix this.

Reply:
I would appreciate some context.

Reply:
I don't know enough about what is going on here to say if this is the best answer, but 
line 46 of Card.vue could be set to false. 
```
    this.mutationObserver = new MutationObserver(this.setHeight);
    this.mutationObserver.observe(this.$refs.wrapper, {
      attributes: true,
      childList: true,
      subtree: false, //change here
      attributeFilter: ['style'],
    });
  },
```

Reply:
The main reason for the mutation observer is so that in case the content of the card changes, the card can resize to the appropriate height. In this case, I'm also emitting the `height-change` event, so that the parent Schedule Card can scroll to the current period. This functionality is only really used in one scenario:
 - When the user switches to a different day, and clicks on the `Go Live` button, in which case the content of the Schedule Card is replaced and therefore, the content is scrolled to the top by default
 - With this functionality, the content changing would trigger the `height-change` event, which will scroll the content back down to the current period

Here's an example:
https://user-images.githubusercontent.com/22779056/128804723-455f36b9-636d-42f9-8676-c83d66a40210.mp4
- notice how at the end, the Schedule Card is scrolled down to period 8

I thing changing `subtree` to false would break this behavior, but I have an idea for an alternate approach, let me try it out